src/validators/bartio/defaultValidatorList.json

```diff
@@ -512,6 +512,14 @@
         "website": "https://www.h2o-nodes.com",
         "twitter": "https://www.twitter.com/H2oNodes"
     },
+    {
+      "id": "0x201c74de018aa34214BEEFFaF717e1d5F36261a7",
+      "logoURI": "https://raw.githubusercontent.com/nodeinfra/publicinfo/main/nodeinfra_logo_256.png",
```

Comment:
Update to: "https://res.cloudinary.com/duv0g402y/raw/upload/src/assets/nodeinfra.png"
We handle the upload from assets folder automatically

Reply:
Done